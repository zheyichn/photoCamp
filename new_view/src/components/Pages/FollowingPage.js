import React from "react";
import Header from "../Activity/Header";
import { Grid } from "@mui/material";
import Following from "../Follow/Following";
import { lightGridStyle } from "../../styles/gridStyles";

export default function FollowingPage() {
  return (
    <div>
      <Header />
      <Grid container flexDirection="column" sx={lightGridStyle}>
        <Grid item width="50%" alignSelf="center" alignItems="center" mt={5}>
          <Following />
        </Grid>
      </Grid>
    </div>
  );
}
