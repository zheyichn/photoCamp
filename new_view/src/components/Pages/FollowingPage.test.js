import { render, screen, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";
import FollowingPage from "./FollowingPage";
import renderer from "react-test-renderer";

afterEach(() => {
  cleanup();
});

describe("test following page", () => {
  it("Create following page body renders correctly", async () => {
    const tree = renderer.create(<FollowingPage />).toJSON();
    await expect(tree).toMatchSnapshot();
  });
});
