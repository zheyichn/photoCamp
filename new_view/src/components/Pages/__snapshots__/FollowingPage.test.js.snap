// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test following page Create following page body renders correctly 1`] = `
<div>
  <div>
    <div
      className="MuiGrid-root css-54nrqh-MuiGrid-root"
    >
      <div
        className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      >
        <div
          className="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
          style={
            {
              "backgroundColor": "white",
            }
          }
        >
          <div
            className="MuiGrid-root MuiGrid-item css-fom513-MuiGrid-root"
            style={
              {
                "backgroundColor": "white",
              }
            }
          >
            <div
              className="MuiTypography-root MuiTypography-h6 css-fd7qc2-MuiTypography-root"
            >
              PhotoCamp
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-container css-raykiu-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item css-1ajuotd-MuiGrid-root"
    >
      <div
        className="friend-list"
      >
        <div />
      </div>
    </div>
  </div>
</div>
`;
